.license

```diff
@@ -0,0 +1,4 @@
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
```

Comment:
Always make sure newlines at end of files are properly there.
You can adjust your IDE here as per our internal guidelines.